.build/.k.rev

```diff
@@ -1 +1 @@
-b3c558f7aadff0db612263c18ec167639ce1b243
+1768d7f317dd00bbd149995fe62203df68ba869d
```

Comment:
which text editor are you using?  no EOL doesn't look good.

Reply:
VS Code. I am not sure why there is no EOL